library/cpp/skiff/skiff.cpp

```diff
@@ -389,6 +433,20 @@ void TUncheckedSkiffWriter::WriteUint128(TUint128 value)
     WriteSimple<ui64>(value.High);
 }
 
+void TUncheckedSkiffWriter::WriteInt256(const TInt256& value)
+{
+    for (int partIndex = 0; partIndex < 4; ++partIndex) {
```

Comment:
value.Parts.size()? Or maybe `for (const auto& part : value.Parts)`

Reply:
No ref because ui32, but sure.